modules/openapi-generator/src/main/resources/JavaSpring/apiRoute.mustache

```diff
@@ -0,0 +1,16 @@
+/**
+* NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech) ({{{generatorVersion}}}).
+* https://openapi-generator.tech
+* Do not edit the class manually.
+*/
+package {{package}};
+
+{{#operations}}
+public interface {{classname}}Route {
```

Comment:
Constants should not be defined in interfaces : https://rules.sonarsource.com/java/tag/bad-practice/RSPEC-1214